ml-proto/spec/check.ml

```diff
@@ -134,9 +134,9 @@ let rec check_expr c et e =
     check_expr_opt c (label c x) eo e.at
 
   | Br_if (x, eo, e) ->
+    check_expr_opt c et eo e.at;
```

Comment:
@rossberg-chromium Ok, so I guess that PR would subsume these others.  The original proposal didn't have `br_if`, but the reduce would have `br_if` return its value, right?